xs/src/libslic3r/Point.cpp

```diff
@@ -18,6 +18,12 @@ Point::operator==(const Point& rhs) const
     return this->coincides_with(rhs);
 }
 
+bool
+Point::operator<(const Point& rhs) const
+{
+    return this->x < rhs.x || this->y < rhs.y;
```

Comment:
Even better, actually, It was not needed. I just left it when deciding to go for another solution. :)